data/org.freedesktop.portal.ProxyResolver.xml

```diff
@@ -41,11 +46,53 @@
         proxy uri are of the form ``protocol://[user[:password] AT host:port``.
         The protocol can be ``http``, ``rtsp``, ``socks`` or another proxying
         protocol. ``direct://`` is used when no proxy is needed.
+
+        This is a convenience wrapper over reading the PAC file or the other
+        portal settings, in that order of priority from highest to lowest.
     -->
     <method name="Lookup">
       <arg type="s" name="uri" direction="in"/>
       <arg type="as" name="proxies" direction="out"/>
     </method>
+    <!--
+    The following values are valid:
+      0: Proxy should not be used.
+      1: Automatic proxy mode; use pac_url.
+      2: Manual proxy mode; read associated settings manually.
+
+      This property was added in version 2.
+    -->
+    <property name="use_proxy" type="u" access="read"/>
+    <!--
+      This property was added in version 2.
+    -->
+    <property name="pac_url" type="s" access="read"/>
```

Comment:
This isn't enough to implement GNOME proxy settings because `pac_url` is optional in GNOME, to allow web proxy autodiscovery. You could use a maybe type `ms` to allow for this.